test/tests/meshgenerators/block_deletion_generator/tests

```diff
@@ -1,12 +1,20 @@
 [Tests]
-  [./block_deletion_test]
+  [block_deletion_test1]
     type = 'Exodiff'
     input = 'block_deletion_test1.i'
     exodiff = 'block_deletion_test1_out.e'
     recover = false
-
     requirement = 'The system shall be capable of deleting all of the elements in a mesh based on subdomain ID.'
     design = 'meshgenerators/BlockDeletionGenerator.md'
     issues = '#11640'
-  [../]
+  []
+  [block_deletion_test13]
+    type = 'Exodiff'
+    input = 'block_deletion_test13.i'
+    exodiff = 'block_deletion_test13_out.e'
+    recover = false
+    requirement = 'The system shall be capable of deleting multiple blocks simultaneously in a mesh based on subdomain ID.'
+    design = 'meshgenerators/BlockDeletionGenerator.md'
+    issues = '#11640 #17052'
+  []
```

Comment:
Should be able to use requirement = and multiple details for this, while also giving the tests a more useful name.